content/1.7/back-office/login-listing.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Login into Back-Office
+weight: 1
+---
+# LOGIN
+
+[Login UI](static/login.png)
+
+This UI is the main feature to authorize the Prestashop Back-Office administrator, and to log-in to administrate and maintain all the Prestashop Back-Office  features and functionalities. This page contains only one main element - the log in form.
+
+## Top UI Section
+
+- The top section represents the Prestashop main title name and the Prestashop version, that the administrator is accessing.
+
+## Middle UI Section
+
+- **Logo** - the official Preston seabird logo of Prestashop.
+- **Shop name** - the shop name, that was defined during the installation process of Prestashop application.
+- **Email address input** - the specific superadmin profile email input text field. The email is the exact that was defined during the installation process of Prestashop. It must be used at the first log-in attempt. 
+- **Password input** - the specific superadmin profile password input text field. The password is the exact that was defined during the installation process of Prestashop. It must be used at the first log-in attempt. 
+- **Login CTA button** - the button, that executes the logging in action, the user then is instantly redirected to the default main dashboard page.
```

Comment:
It is missing the error cases. 
What happens if the user doesn't use an email from a SuperAdmin or an Employee account?
What happens if the user doesn't use a password from a SuperAdmin or an Employee account?
Does the form warn the user if the email is formatted as an email?

Reply:
> What happens if the user doesn't use an email from a SuperAdmin or an Employee account? What happens if the user doesn't use a password from a SuperAdmin or an Employee account? 
It is always the same message `The employee does not exist, or the password provided is incorrect.`

> Does the form warn the user if the email is formatted as an email?

Yes, instant red error message appears, about email, that it is not valid